packages/ui-library/src/components/internal-components/form-caption/index.css.ts

```diff
@@ -20,7 +20,6 @@ export const { tokenizedLight: captionLight, tokenizedDark: captionDark } = rend
         gap: ${CaptionComponent.Container.ItemSpacing.SM};
 
         .blr-icon {
-          padding-top: ${CaptionComponent.IconWrapper.PaddingTop.SM};
```

Comment:
@larserbach opinion?

Reply:
The removal of this line seems wrong.
The intention of this padding-top is to align the icon against the first line of text.
But it might be important to understand, that by design, the icon is wrapped inside an "iconWrapper". The padding is assigned on "iconWrapper" not the icon itself. (I believe we should not mess with the css of included components if possible).
<img width="1470" alt="image" src="https://github.com/deven-org/B01LER-Kitchen/assets/1247117/2d61ac3b-af35-4404-be4a-4a06534a8fe1">